daq/generator.py

```diff
@@ -40,13 +41,58 @@ def _normalize_raw_topo(self):
         topo_dir = self.config.get('topo_dir', 'inst')
         if not os.path.exists(topo_dir):
             os.makedirs(topo_dir)
+        configs = {}
         for filename in os.listdir(source_dir):
             if not filename.endswith(self._YAML_POSTFIX):
                 LOGGER.info('Skipping non-yaml file %s', filename)
                 continue
             in_path = os.path.join(source_dir, filename)
-            loaded_yaml = self._load_config(in_path)
-            self._write_yaml(topo_dir, filename, loaded_yaml)
+            configs[filename] = self._load_config(in_path)
+        self._flatten_configs(configs)
+        key_tracker = set(configs.keys())
+        for config in configs:
+            self._write_yaml(topo_dir, config, configs[config])
+            key_tracker.remove(config)
+        for config in key_tracker:
+            LOGGER.warning('Unused key %s', config)
```

Comment:
I am not sure we'll ever reach this line, considering we are getting rid of all keys in the for loop. Perhaps we dont need "key_tracker" either?

Reply:
Removed.